src/index.js

```diff
@@ -337,7 +337,9 @@ export default function (schema, opts) {
       doc = doc.value
     }
 
+    if (!doc) return next()
```

Comment:
I want to discuss on this fix.

This basically means `if(this.options.new && !doc) return next()` since in line [332](https://github.com/codepunkt/mongoose-patch-history/pull/98/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R332) it is guaranteed that `this.options.new` is true.

However this workaround if `this.options.new` is true, is not needed anymore.
If we just instantly `return postUpdateOne.call(this, {}, next)` everything is working as expected.

This depends on line [362](https://github.com/codepunkt/mongoose-patch-history/pull/98/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R362) as there the document is checked for existance.

My proposal would therefore be to delete the whole workaround and instantly return `postUpdateOne()`.

Reply:
Sounds good, it makes the code more readable.